docs/administration/configuration/configuration.md

```diff
@@ -36,6 +36,10 @@ These files contain additional settings and point to the general (not environmen
 
     Read more about [how configuration is handled in Symfony]([[= symfony_doc =]]/best_practices/configuration.html).
 
+!!! caution
+
+    Avoid using special characters in your configuration files, avoid especially Unicode characters from the ["Other" (`C`) categories (such as control or format characters)](https://en.wikipedia.org/wiki/Unicode#General_Category_property). Make sure your IDE displays them. Be careful when copy-pasting from a word processing software or a PDF (it might content hidden character like [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen).
```

Comment:
```suggestion
    Avoid using special characters in your configuration files. More specifically, do not use Unicode characters from the ["Other" (`C`) categories](https://en.wikipedia.org/wiki/Unicode#General_Category_property), such as control or format characters. 
    Make sure your IDE displays them. 
    
    Be careful when copy-pasting text from a word processing software or a PDF, because it might contain hidden characters like the [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen).
```

Is this a recommendation, or a prohibition? If the latter, I would go with "do not use", not "avoid". Same goes to all instances below

Reply:
"Don't" instead of "do not"

Reply:
```suggestion
    Avoid using special characters in your configuration files. More specifically, don't use Unicode characters from the ["Other" (`C`) categories](https://en.wikipedia.org/wiki/Unicode#General_Category_property), such as control or format characters.

    Make sure your IDE displays them. 

    Be careful when copy-pasting text from a word processing software or a PDF, because it might contain hidden characters like the [soft hyphen](https://en.wikipedia.org/wiki/Soft_hyphen).
```